src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx

```diff
@@ -1,35 +1,167 @@
 ---
-title: Monitor your Capacitor application
+title: Monitor your Ionic Capacitor application
 tags:
   - Mobile monitoring
   - New Relic Mobile Capacitor
   - Get started
-metaDescription: "New Relic's Capacitor agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
+metaDescription: "New Relic's Ionic Capacitor agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
 redirects:
   - /docs/mobile-monitoring/new-relic-mobile-capacitor/get-started
 ---
 
 import mobileQueryExample from 'images/mobile_screenshot-full_query-example.webp'
 
-Our New Relic Capacitor agent monitors your Capacitor mobile app and provides deep insights into your app's performance, errors, and user experience. Written in JavaScript, the plugin automatically includes New Relic's native agents to provide mobile monitoring and performance visibility. Once you install and configure the Capacitor agent, you'll be able to:
+Our New Relic Ionic Capacitor agent monitors your Ionic Capacitor mobile app and provides deep insights into your app's performance, errors, and user experience. Written in JavaScript, the plugin automatically includes New Relic's native agents to provide mobile monitoring and performance visibility. Once you install and configure the agent, you'll be able to:
```

Comment:
Again, if there's no reason to mention JavaScript then we should cut that part, but if there is we should keep it.